src/modules/number/index.ts

```diff
@@ -534,4 +534,165 @@ export class NumberModule extends SimpleModuleBase {
 
     return result;
   }
+
+  /**
+   * Generates a random number between `min` and `max` using an exponential distribution.
```

Comment:
```suggestion
   * Generates a random float between `min` and `max` using an exponential distribution.
```

Emphasise its a float NOT an int (since the tables below might mislead people if they dont notice the values have been `floor`ed)

Reply:
Would using `Math.floor(result)` as a table header solve this issue as well?
(I'm not against changing number to float)

Reply:
Or you could show ranges in the table like "0 to 1" or "0  <= x < 1"?

Reply:
Done.